src/plugins/performance.ts

```diff
@@ -35,10 +30,10 @@ class PerformanceProfiler implements IComponent {
 
   public activate() {
     if (this.timing) {
-      this.logTimingTimeout = setTimeout(this.logTiming.bind(this), timeoutLength);
+      this.logTimingTimeout = setTimeout(this.logTiming.bind(this), this.timeoutLength);
```

Comment:
Once we move over to supporting configuration per plugin. Probably going to be out of scope for this PR though unless you feel strongly about it.